text-ui-test/EXPECTED.TXT

```diff
@@ -7,20 +7,15 @@ Hello from
 \_____|______|_____|_|  \___|\__,_|_|\__| /_/    \_\___/___/_|___/\__\__,_|_| |_|\__|
 What is your name?
 Hello struggling CG1111 student
-You have entered Tutorial Mode!
-Type 'exit' if you want to leave this mode and go back to the application.
-Lets get started! First choose a template you want to work with.You can choose from rc, rl, lc and r templates.
-eg. To select the Resistor-Capacitor circuit template, enter 'template rc.'
+Invalid Command!
```

Comment:
Should not be invalid command, need to change `input.txt` to use `tutorial circuit` instead.

Reply:
Yup yup had mentioned when i created the PR:
> Have to update runtest.bat with apt commands.